docs/src/main/tut/docs/index.md

```diff
@@ -34,7 +34,7 @@ To create an instance of the HTTP client, use the `ElasticClient` companion obje
 Requests are created using the elastic4s DSL. For example to create a search request, you would do:
 
 ```scala
-search("index" / "type").query("findthistext")
+searchIndex("index").query("findthistext")
```

Comment:
It's just search not search index I think.

Reply:
Fixed!